### PR TITLE
apt-get cheat sheet: added more commands

### PR DESCRIPTION
**Pull request by mayank7**
Added more commands to apt-get cheat sheet

IA Page: https://duck.co/ia/view/apt_get_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
## [AptGet Cheat Sheet](https://duck.co/ia/view/apt_get_cheat_sheet)

**Description**: My instant answer provides the most commonly used commands of the AptGet package manager for Debian based GNU/Linux systems.

**Example Query**: [Apt get cheat sheet](https://beta.duckduckgo.com/?q=Apt get cheat sheet), [](https://beta.duckduckgo.com/?q=)

**Tab Name**: Answer

**Source**:  

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/apt_get_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/apt_get_cheat_sheet)*

File: share/goodie/cheat_sheets/json/apt-get.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
         "sourceUrl": "https://help.ubuntu.com/community/AptGet/Howto"
     },
     "aliases": [
-        "aptget", "apt-get"
+        "aptget", "apt-get", "apt get"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/apt-get.json]: Remove this alias.

Comment (mayank7) [inline share/goodie/cheat_sheets/json/apt-get.json]: @GuiltyDolphin I removed it.

**Comment by mintsoft** [comment]
@mayank7 this PR has conflicts now, can you rebase?

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

**Pull request merged**